code/modules/multiz/movement.dm

```diff
@@ -49,6 +49,25 @@
 			else
 				to_chat(src, "<span class='warning'>You gave up on pulling yourself up.</span>")
 				return 0
+		else if(ismob(src)) //VOREStation Edit Start. Are they a mob, and are they currently flying??
+			var/mob/H = src
+			if(H.flying == 1)
+				if(H.incapacitated(INCAPACITATION_ALL))
+					to_chat(src, "<span class='notice'>You can't fly in your current state.</span>")
+					H.stop_flying() //Should already be done, but just in case.
+					return 0
+				var/fly_time = max(7 SECONDS + (H.movement_delay() * 10), 1) //So it's not too useful for combat. Could make this variable somehow, but that's down the road.
+				to_chat(src, "<span class='notice'>You begin to fly upwards...</span>")
+				destination.audible_message("<span class='notice'>You hear the flapping of wings.</span>")
+				H.audible_message("<span class='notice'>[H] begins to flap \his wings, preparing to move upwards!.</span>")
```

Comment:
There's a stray period after the `!` here.

Reply:
Good catch.